src/checks.jl

```diff
@@ -47,24 +47,47 @@ function check_data(case, modeltype::EnergyModel)
     global logs = []
     log_by_element = Dict()
 
+
+    if !check_timeprofiles
+        @warn "Checking of the time profiles is deactivated:\n" *
+        "Deactivating the checks for the time profiles is strongly discouraged.\n" *
```

Comment:
This warning seems longer than needed. The two first sentences should be sufficient.

Reply:
I am aware that the warning is quite long. It is however only printed once when running and it is important that it is not getting lost above the solver information coming in. Hence, it will remain.

Reply:
Also it would be nice to remove the \n so the message can fit on the same line.

Reply:
When running the model with SDDP it runs once for each node.

Reply:
I was now included that the warning is shown only once. In addition, the newlines (`\n`) were adjusted to only occur were it makes sense.